src/components/AttachmentCarousel/index.js

```diff
@@ -292,7 +303,8 @@ class AttachmentCarousel extends React.Component {
 
                         // Enable scrolling by swiping on mobile (touch) devices only
                         // disable scroll for desktop/browsers because they add their scrollbars
-                        scrollEnabled={this.canUseTouchScreen}
+                        // disable scroll in PDF zoomed state
```

Comment:
the comment says that we disable scroll on zoomed pdfs, but the code seems to disable it when it's not zoomed. Can we make this consistent?

Reply:
The code enables scroll (scrolling the FlatList with different attachments horizontally) when we are not in a zoomed state. And disables scroll (changing attachments) in a zoomed state. There could be a misunderstanding on what is to scroll in this context.